subsys/bluetooth/mesh/scene_srv.c

```diff
@@ -676,6 +677,14 @@ static int scene_srv_init(struct bt_mesh_model *model)
 	sys_slist_prepend(&scene_servers, &srv->n);
 
 	srv->model = model;
+
+	srv->dtt_srv = bt_mesh_dtt_srv_get(bt_mesh_model_elem(model));
```

Comment:
Probably there is no need in `dtt_srv` in this case as it can be obtained at any point.